global.css

```
@@ -46,16 +48,15 @@ h1 {
 main {
   display: flex;
   flex-direction: column;
-  justify-content: center;
   align-items: center;
-  padding: 150px 100px 150px 100px;
+  padding: 150px 0 150px 0;
```

Here you can use the shorthand notation (same result)
```suggestion
  padding: 150px 0;
```